File: Aprimoramento Computacional/23-09/A.py
from math import floor

N = int(input()) # número de diretores
K = int(input()) # duração máxima da reunião (em minutos)

tempoSemPausa = K - floor(N / 2)
tempoDeFala = floor(tempoSemPausa / N)

print(tempoDeFala)

File: Aprimoramento Computacional/23-09/A.c
#include <stdio.h>
#include <math.h>

int main() {
    int N;
    int K;

    scanf("%d", &N);
    scanf("%d", &K);

    N = N / 2;
    printf("%d", N);
    printf("%d", floor(N));

    return 0;
}
